cloud/blockstore/tools/csi_driver/internal/driver/node.go

```diff
@@ -687,17 +687,62 @@ func logVolume(volumeId string, format string, v ...any) {
 	log.Printf("[v=%s]: %s", volumeId, msg)
 }
 
+func (s *nodeService) doesVolumeExist(
```

Comment:
better name could be `hasVolume` or `volumeExists`

Reply:
I decided to remove this function(and similar function in controller.go). it returns true/false however it's a three possible states in fact: volume exists, volume doesn't exist and unknown.
It's a bit more code now but handling of the error is not subtle anymore